scripts/travisci/check_flake8.sh

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env bash
+# Exit status of this script
+exit_status=0
+
+# Add an option to run the incremental verification locally to reduce the time
+# spent in travis for error verification. It's required to rebase the feature
+# branch with the master branch to avoid checking on unknown changes.
+while getopts ":l" opt; do
+  case ${opt} in
+    l)
+      TRAVIS_PULL_REQUEST="false"
+      ;;
+    \?)
+      echo "usage: check_flake8 [-l]"
+      echo "    -l    run incremental verification locally"
+      exit 1
+      ;;
+  esac
+done
+
+# ABSOLUTE VERIFICATION:
+# The following errors are analyzed in all python files across the repository.
+#
+# TABS
+# W191 indentation contains tabs
+# E101 indentation contains mixed spaces and tabs
+errors="E101,W191,"
```

Comment:
Please split this into errors_incremental and errors_absolute for ease of maintenance. You can concatenate them when when you call the script.

Reply:
Done.